go/libraries/doltcore/sqle/tables.go

```diff
@@ -2705,12 +2707,14 @@ func (t *WritableDoltTable) createForeignKey(
 	if ok {
 		refTableIndexName = refTableIndex.Name()
 	}
+	
+	// TODO: foreign key defn should include schema names
```

Comment:
It took me a little bit to get that `defn` is `definition`, seems like a weird shorthand. Perhaps from before my time 😆

Reply:
Hmm, will keep that in mind. Comment will be gone later today